static/src/stylesheets/module/site-messages/_engagement-banner.scss

```diff
@@ -52,12 +50,15 @@
 }
 
 .engagement-banner__text {
-    color: $brightness-7;
+    @include banner-copy-alignment;
 
-    font-size: 17px;
-    line-height: 22px;
+    color: $brightness-7;
+    font-size: 14px;
+    line-height: 18px;
```

Comment:
for a separate pr: can we make these be any of the font size mixins we already have in place? @zeftilldeath can probably help us with figuring out the closest one